compiler/circle-mpqsolver/src/core/TestHelper.test.cpp

```diff
@@ -208,5 +208,36 @@ bool isFileExists(const std::string &path)
 }
 
 } // namespace io_utils
+
+namespace data_utils
+{
+
+class SampleDataInput final : public core::DataProvider
+{
+public:
+  SampleDataInput() : _num_of_samples(1) {}
+  size_t numSamples() const override { return _num_of_samples; }
+  uint32_t numInputs(uint32_t sample) const override { return 1; }
+  void getSampleInput(uint32_t sample, uint32_t input, core::InputData &data) const
+  {
+    size_t size = data.data().size() / sizeof(float);
+    auto floats = reinterpret_cast<float *>(data.data().data());
+    for (uint32_t idx = 0; idx < size; idx++)
+    {
+      floats[idx] = 0.f; // or any other value
+    }
+  }
+
+private:
+  uint32_t _num_of_samples;
```

Comment:
I don't know much about `DataProvider`,
if `SampleDataInput` only provides single sample, then `numInputs` uses `1` not `_num_of_samples` the `_num_of_samples` seems useless.

Reply:
Ahh. Yes. Sorry. You're right. I'll remove it.